support/package_jq

```diff
@@ -5,12 +5,15 @@ source "$basedir/../conf/buildpack.conf"
 source "$basedir/lib/utils"
 source "$basedir/lib/swift"
 
+jq_download_url="https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux64"
```

Comment:
Should we make this configurable via the env variable `JQ_DOWNLOAD_URL` so that if it changes in the future we can update it via the env? 

```suggestion
jq_download_url="${JQ_DOWNLOAD_URL:-https://github.com/jqlang/jq/releases/download/jq-1.7/jq-linux64}"
```

Reply:
Done in the latest commit. Thanks for the idea :)